x/auth/client/cli/tx_sign.go

```diff
@@ -270,10 +282,10 @@ func makeSignCmd() func(cmd *cobra.Command, args []string) error {
 	}
 }
 
-func getSignatureJSON(cdc codec.JSONMarshaler, newTx types.StdTx, generateSignatureOnly bool) ([]byte, error) {
+func getSignatureJSON(cdc codec.JSONMarshaler, txGen client.TxConfig, txBldr client.TxBuilder, newTx sdk.Tx, generateSignatureOnly bool) ([]byte, error) {
 	if generateSignatureOnly {
-		return cdc.MarshalJSON(newTx.Signatures[0])
+		return cdc.MarshalJSON(txBldr.GetTx().GetSignatures())
```

Comment:
I don't think we should do this actually. I think the best workaround is to use `StdSignature` for now because we don't have the right thing in protobuf yet.